src/datatypes/exercise001/exercise001.java

```diff
@@ -0,0 +1,21 @@
+package datatypes.exercise001;
+
+import java.util.Scanner;
+
+/***
+ * Write a Java program to convert temperature from Fahrenheit to Celsius degree.
+ */
+public class exercise001 {
+    public static void main(String[] args) {
+        Double fahrenheitTemperature, celsiusTemperature;
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Input temperature in Fahrenheit: ");
+        fahrenheitTemperature = scanner.nextDouble();
```

Comment:
Jeśli to możliwe, to dobrą praktyką jest robienie inicjalizacji i deklaracji w jednej linii.
raczej :Double fahrenheitTemperature = scanner.nextDouble();
Double celsiusTemperature = getCelsiusTemperature(fahrenheitTemperature);